app/react/kubernetes/components/KubernetesEventsDatatable/columns/message.tsx

```diff
@@ -0,0 +1,6 @@
+import { columnHelper } from './helper';
+
+export const message = columnHelper.accessor('message', {
+  header: 'Message',
+  cell: ({ row: { original: event } }) => event.message,
```

Comment:
that's almost the default cell function. The default being `({getValue}) => getValue()`, and results with the same output.
```suggestion
```